fxtest/app_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package fxtest
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/fx"
+)
+
+func TestApp(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		spy := newTB()
+
+		New(spy).RequireStart().RequireStop()
+
+		assert.Zero(t, spy.failures, "App didn't start and stop cleanly.")
+		assert.Contains(t, spy.logs.String(), "running", "Expected to write logs to TB.")
+	})
+
+	t.Run("NewFailure", func(t *testing.T) {
+		spy := newTB()
+
+		New(
+			spy,
+
+			// Missing string dependency in container.
```

Comment:
Fix formatting?

Reply:
What would you like changed?

(This is correctly formatted as far as gofmt is concerned.  `gofmt -l fxtest/ app_test.go` is empty.)